docs/configuration.md

```diff
@@ -0,0 +1,541 @@
+
+```csharp
+// This mutation will be skipped
+ConfigureAwait(true); 
+
+// This mutation won't because we cannot currently detect this
+var t = true;
+ConfigureAwait(t); 
+```
+
+You can also ignore constructors by specifying the type and adding the `.ctor` suffix.
+
+Both, method names and constructor names support wildcards.
+
+```json
+"['*Log']" // Ignores all methods ending with Log
+"['*Exception.ctor']" // Ignores all exception constructors
+```
+
+## Optimization
+
+### `coverage-analysis` <`string`>
+
+Default: `perTest`  
+Command line: `N/A`  
+Config file: `"coverage-analysis": 'off'`
+
+Use coverage info to speed up execution. 
+
+- **perTest**: capture the list of mutants covered by each test. For every mutant that has tests, only the tests that cover the mutant are used to test a mutant. Mutants without tests are reported as `NoCoverage`. Fastest option.
+- **perTestInIsolation**: like 'perTest', but running each test in an isolated run. This results in more accurate
+coverage information for some mutants (see below), at the expense of a longer startup time.
+- **all**: capture the list of mutants covered by a test. Test only the mutants covered by unit tests. Non covered mutants are assumed as survivors. Fast option.
+- **off**: coverage data is not captured. All unit tests are ran against all mutants.
+
+#### Notes on coverage analysis
+* Results should not be impacted by coverage analysis. If you identify a suspicious survivor, run
+Stryker again without coverage analysis and report an issue if this mutant is killed by this run.
+* when using `perTest` mode, mutants that are executed as part as some static constructor/initializer 
+are run against all tests as Stryker cannot reliably capture coverage for those. This is a consequence of static
+constructors/initialisers being called only once during tests. This heuristic is not needed when using
+`perTestInIsolation` due to test being run one by one.
+
+### `disable-bail` <`bool`>
+
+Default: `false`  
+Command line: `N/A`  
+Config file: `"disable-bail": true`
+
+Stryker aborts a unit testrun for a mutant as soon as one test fails because this is enough to confirm the mutant is killed. This can reduce the total runtime but also means you miss information about individual unit tests (eg if a unit test does not kill any mutants and is therefore useless). You can disable this behavior and run all unit tests for a mutant to completion. This can be especially useful when you want to find useless unit tests.
+
+### `disable-mix-mutants` <`bool`>
+
+Default: `false`  
+Command line: `N/A`  
+Config file: `"disable-mix-mutants": true`
+
+Stryker combines multiple mutants in the same testrun when the mutants are not covered by the same unit tests. This reduces the total runtime. You can disable this behavior and run every mutation in an isolated testrun. This can be useful when mixed mutants have unintended side effects.
+
+### `since` <`flag`>
```

Comment:
I think all the since features could use its own heading

Reply:
I tried making the headings either categories (ie optimizations) or the options itself options. Adding another level of heading would be too much imo. The since options are already grouped in the optimizations category and I do not believe they are a category in and of themselves.